addons/pom.xml

```diff
@@ -16,11 +17,13 @@
   <name>openHAB Add-ons</name>
 
   <modules>
+    <!--
     <module>binding</module>
     <module>extensionservice</module>
     <module>io</module>
     <module>persistence</module>
     <module>transform</module>
+    -->
```

Comment:
Shouldn't these modules be uncommented again? 
I think they were commented to speed up testing? :-)